### PR TITLE
Podcasts: Add 241 recap

### PR DESCRIPTION
**Pull request by bitschmidty**
Adds podcast for newsletter 241. The podcast audio is live on the RSS feed. Awaiting transcription.

### CODE REVIEW

**Comment by bitschmidty** [comment]
Put in the transcription and associated anchors.

File: _posts/en/podcast/2023-03-09-newsletter-recap.md [hunk 1]
```diff
@@ -0,0 +1,808 @@
...
+this little backed-up data and right after the handshake.  So that way, if you
+lose your channel state, when the channels are closed, they go to the predefined
+payout addresses, and that's included in the backup that every single peer of
+yours has.  And because the peers give you these, whatever, 50 kB every single
+time you reconnect to them, you don't really have to ask them for it and make
+them aware that you might have lost your channel state, but they just give it to
+you automatically and you know who your good peers are that way, and you always
+regain access to your static channel backup.
+
+So, this is notably not default backup of the states of the channel, but it's
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-09-newsletter-recap.md:713]:
```suggestion
So, this is notably not the full backup of the states of the channel, but it's
```

File: _posts/en/podcast/2023-03-09-newsletter-recap.md [hunk 2]
```diff
@@ -0,0 +1,808 @@
...
+
+**Mike Schmidt**: Okay, I couldn't see that but perhaps they are adopting that
+new BIP fairly soon after it was official.
+
+{:#lnd}
+_LND v0.16.0-beta.rc2_
+
+The last release we have for this week is LND v0.16.0-beta.rc2.  I didn't pull
+out anything notable in this release to discuss.  You're welcome to look at the
+release notes about this rc.  Merch, I'm not sure if there's anything that you
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-09-newsletter-recap.md:757]:
```suggestion
release notes about this RC.  Mrch, I'm not sure if there's anything that you
```

**Review by Xekyo — COMMENTED** [review]
Skimmed it, just a few things that jumped out to me

**Pull request merged**